modules/private-clusters-about-ibm-cloud.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_ibm_cloud_public/installing-ibm-cloud-private.adoc
+
+:_content-type: CONCEPT
+[id="private-clusters-about-ibm-cloud_{context}"]
+= Private clusters in IBM Cloud VPC
+
+To create a private cluster on IBM Cloud VPC, you must provide an existing private VPC and subnets to host the cluster. The installation program must also be able to resolve the DNS records that the cluster requires. The installation program configures the Ingress Operator and API server for only internal traffic.
+
+The cluster still requires access to internet to access the IBM Cloud VPC APIs.
+
+The following items are not required or created when you install a private cluster:
+
+* Public subnets
+* Public network load balancers, which support public ingress
+* A public DNS zone that matches the `baseDomain` for the cluster
+
+The installation program does use the `baseDomain` that you specify to create a private DNS zone and the required records for the cluster. The cluster is configured so that the Operators do not create public records for the cluster and all cluster machines are placed in the private subnets that you specify.
+
```

Comment:
1. the DSN zone need be prepared before.
2.  installation program just bind with the network  of VPC which specified in install-config.

Reply:
Correct. We now clearly state the need for users to create a DNS zone in:

- Configuring DNS resolution > [Using IBM Cloud DNS Services for DNS resolution](https://50561--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_cloud_public/installing-ibm-cloud-account.html#installation-dns-ibm-cloud_installing-ibm-cloud-account)
- [Private clusters](https://50561--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_cloud_public/installing-ibm-cloud-private.html#private-clusters-default_installing-ibm-cloud-private)